April 2025 newsletter

twinkarma opened the pull request:
I've put the publish date as 9th of April but can release it early if we can finalise it by tomorrow.

Robadob commented on _newsletters/2025-04-09-newsletter.md:
```diff
@@ -0,0 +1,209 @@
+  - 2025-06-11: Open data, indigenous data sovereignty and the CARE principles
+- 2025-05-14 \- 2025-05-15: [Symposium on sustainable digital futures](https://digitalgood.net/call-for-papers-towards-sustainable-digital-futures-a-two-day-symposium/)
+- 2025-05-19 \- 2025-05-21: [Pint of Science Sheffield](https://pintofscience.co.uk/events/sheffield)
+- 2025-05-30: [OpenFest call for proposal deadline](https://www.sheffield.ac.uk/openresearch/events/openfest-2025)
```
Is it worth including the date of openfest too as a separate item?

twinkarma replied:
Added in the new commit

Robadob commented on _newsletters/2025-04-09-newsletter.md:
```diff
@@ -0,0 +1,209 @@
+- 2025-04-07: [2025 RSE Midlands Annual Conference, Birmingham](https://rse-midlands.github.io/docs/event-7th-april-2025/)
+- 2025-04-15: [EVERSE Webinar: Building the Landscape of Software Quality and Defining Software Quality Indicators](https://indico.cern.ch/event/1531998/)
+- 2025-05-05 \- 2025-05-08: [Sustainable Scientific Software Conference (S3C) [Denver, Colorado, USA]](https://s3c.sandia.gov/)
+- 2025-05-13 \- 2025-05-15: [Collaborations Workshop 2025 at the University of Stirling](https://www.software.ac.uk/workshop/collaborations-workshop-2025-cw25).
```
```suggestion
- 2025-05-13 \- 2025-05-15: [SSI Collaborations Workshop 2025 at the University of Stirling](https://www.software.ac.uk/workshop/collaborations-workshop-2025-cw25).
```

Robadob commented on _newsletters/2025-04-09-newsletter.md:
```diff
@@ -0,0 +1,209 @@
+### Opportunities
+
+#### [Join the 2025 Maintainer Onboarding Cohort!](https://carpentries.org/blog/2025/04/maintainer-onboarding-2025/)
+The Carpentries is recruiting maintainers for several of their lessons.
```
```suggestion
The Carpentries is recruiting maintainers for several of their lessons, deadline 11th April.
```

Robadob commented on _newsletters/2025-04-09-newsletter.md:
```diff
@@ -0,0 +1,209 @@
+
+Funding from the Research Software Maintenance Fund can be used to support and maintain existing research software or sustain a team responsible for multiple software tools. However, it is not intended for developing entirely new software.
+
+The scheme includes two types of awards: up to £500k for two years and up to £150k for one year, with multiple funding rounds between 2025 and 2027. The two year grants will only be available in the first round in 2025.
```
Round 1 expressions of interest are Open April 7th-May 30th

twinkarma replied:
Thanks Rob, I've added the dates in the title

Robadob reviewed (approved):
Added a few minor comments, but looks good at a glance.

twinkarma commented:
Thanks for your review @Robadob, added your suggestions in the latest commit.

ns-rse commented on _newsletters/2025-04-09-newsletter.md:
```diff
@@ -0,0 +1,210 @@
+
+Applicants are asked to demonstrate their engagement in open research in one or more of the following areas:
+
+Publishing research outputs (including data, software, protocols, monographs...) using open licences to promote broad dissemination;
```
Would these read/format better as an unordered list?

twinkarma replied:
Sounds good. Fixed in latest commit.

ns-rse commented on _newsletters/2025-04-09-newsletter.md:
```diff
@@ -0,0 +1,210 @@
+
+Funding from the Research Software Maintenance Fund can be used to support and maintain existing research software or sustain a team responsible for multiple software tools. However, it is not intended for developing entirely new software.
+
+The scheme includes two types of awards: up to £500k for two years and up to £150k for one year, with multiple funding rounds between 2025 and 2027. The two year grants will only be available in the first round in 2025.
```
Should we include an explicit suggestion along the lines of...

```suggestion
The scheme includes two types of awards: up to £500k for two years and up to £150k for one year, with multiple funding rounds between 2025 and 2027. The two year grants will only be available in the first round in 2025.

If you are considering applying and would like RSE involvement and support please get in contact as early as possible by mailing [rse@sheffield.ac.uk](mailto:rse@sheffield.ac.uk) to discuss.
```

ns-rse commented on _newsletters/2025-04-09-newsletter.md:
```diff
@@ -0,0 +1,210 @@
+Using open and participatory methods including open notebooks, co-production, participatory action research and citizen science;
+Disseminating research using accessible outputs such as blogposts, podcasts, animations, infographics, interactive applications and more.
+
+#### [SSI - Software maintenance Fund - Round 1 (7th April - 30th May)](https://www.software.ac.uk/research-software-maintenance-fund/round-1)
```
Minor but matches case in other places.

```suggestion
#### [SSI - Software Maintenance Fund - Round 1 (7th April - 30th May)](https://www.software.ac.uk/research-software-maintenance-fund/round-1)
```

ns-rse reviewed (changes requested):
:+1: Thanks for blasting through this so quickly @twinkarma 

Some minor suggestions from me are noted in-line.

tdjames1 commented on _newsletters/2025-04-09-newsletter.md:
```diff
@@ -0,0 +1,212 @@
+The [FAIR<sup>2</sup> for research software training programme](https://rse.shef.ac.uk/training/fair4rs/) has been developed by the RSE, Data Analytics Service and Library teams and launched in October. The full syllabus is now live and places on upcoming courses can be booked via [MyDevelopment](https://mydevelopment.csod.com/ui/lms-learning-details/app/curriculum/06573f55-12e4-44f1-be4c-fac5ea9 ...
+
+- 2025-04-11, 1.30pm \- 5.00pm [Packaging](https://rse.shef.ac.uk/training/fair4rs/#packaging)
+- 2025-04-29, 1pm \- 2pm: Publishing a software paper in the Journal of Open Source Software
```
```suggestion
- 2025-04-11, 1.30pm \- 5.00pm [Software packaging](https://mydevelopment.csod.com/ui/lms-learning-details/app/event/0cbafddd-4c31-4e8a-bce6-a12e895a4acf)
- 2025-04-29, 12.45pm \- 2pm: [Publishing a software paper in the Journal of Open Source Software](https://mydevelopment.csod.com/ui/lms-learning-details/app/event/9fa9503c-81a1-4667-af7b-14d62a4b82bc) (hybrid, with lunch for registered in-person attendees)
```

tdjames1 commented on _newsletters/2025-04-09-newsletter.md:
```diff
@@ -0,0 +1,212 @@
+Due to capacity within the RSE team, there was very limited LunchBytes content over the past academic year.
+This should be changing soon, a new coordinator for LunchBytes has been found (Research IT's Farhad Allian), with a plan to reboot these lunchtime seminars in the spring semester.
+
+More information will be coming out over the coming months, so sign up to the [RSE mailing list](https://groups.google.com/a/sheffield.ac.uk/g/RSE-group) if you'd like to learn more about research software and associated practices, or get in contact with [Farhad](mailto:farhad.allian@sheffield.ac.uk) if you would like to share what you know at an upcoming session!
```
The 29th April JOSS session is a LunchByte, and I believe there are plans for others in progress - check in with @f-allian?

twinkarma replied:
@f-allian any upcoming luchbytes sessions you'd like to add here?

twinkarma replied:
Thanks @tdjames1 looks like this is all the planned lunchbytes for now.

tdjames1 reviewed (approved):
Amended FAIR4RS info and note about LunchBytes.

twinkarma commented:
Thanks for the review @ns-rse!

twinkarma committed:
April newsletter

Update _newsletters/2025-04-09-newsletter.md

Co-authored-by: Neil Shephard <n.shephard@sheffield.ac.uk>
Update _newsletters/2025-04-09-newsletter.md

Co-authored-by: Neil Shephard <n.shephard@sheffield.ac.uk>
Add suggested fix

Update _newsletters/2025-04-09-newsletter.md

Co-authored-by: T D James <tdjames1@users.noreply.github.com>
Highlight hybrid and lunch for JOSS session

Move event from training to lunchbytes

Change publish date

twinkarma commented:
Thanks for the review @tdjames1

ns-rse commented on _newsletters/2025-04-04-newsletter.md:
```diff
@@ -0,0 +1,211 @@
+- [The Value of Open Source Software](https://www.hbs.edu/faculty/Pages/item.aspx?num=65230), Hoffmann, M. et al.
+- [(Pre-print) No Silver Bullets: Why Understanding Software Cycle Time is Messy, Not Magic](https://doi.org/10.48550/arXiv.2503.05040), John C. et al.
+- [(Pre-print) The Introduction of README and CONTRIBUTING Files in Open Source Software Development](https://doi.org/10.48550/arXiv.2502.18440), Gaughan M. et al.
+
```
```suggestion
- [Evidencing the Impact of Research Software Engineers on Arts & Humanities Scholarship](https://zenodo.org/records/15083632) Beavan D. et al.
- [Towards a National Research Software Engineering Capability in Arts and Humanities Research: a Roadmap](https://zenodo.org/records/15083396) Beavan D. et al.

```

ns-rse reviewed (approved):
Added a couple of RSE Arts & Humanities reports/papers I just clocked.

twinkarma committed:
Update _newsletters/2025-04-04-newsletter.md

Co-authored-by: Neil Shephard <n.shephard@sheffield.ac.uk>
